cmd/uql/execute.go

```diff
@@ -96,7 +96,7 @@ type position struct {
 
 type uqlService interface {
 	Execute(query *Query, apiVersion ApiVersion) (parsedResponse, error)
```

Comment:
If you add **options** to **Continue**, why not also to **Execute**? (see my other comment. I'm against it)

Reply:
Resolved this in the updated client based implementation 

https://github.com/cisco-open/fsoc/blob/e3338c589daffddf0f80facb966729c53d87cad0/cmd/uql/execute.go#L106-L110